parquet/src/file/metadata.rs

```diff
@@ -55,8 +55,8 @@ use crate::schema::types::{
 pub struct ParquetMetaData {
     file_metadata: FileMetaData,
     row_groups: Vec<RowGroupMetaData>,
-    page_indexes: Option<Vec<Index>>,
-    offset_indexes: Option<Vec<Vec<PageLocation>>>,
+    page_indexes: Option<Vec<Vec<Index>>>,
```

Comment:
```suggestion
    /// Page index for all pages in each column chunk
    page_indexes: Option<Vec<Vec<Index>>>,
```
Or something like that, same for the below